pkg/neotest/signer.go

```diff
@@ -27,6 +27,14 @@ type Signer interface {
 	SignTx(netmode.Magic, *transaction.Transaction) error
 }
 
+// Signer is a generic interface which can be either simple-signature signer.
+type SingleSigner interface {
+	Signer
+	// Account returns underlying account which can be used to
+	// get public key and/or sign arbitrary things.
+	Account() *wallet.Account
```

Comment:
I'm not sure we need this additional interface though, maybe `Signer` can have `Accounts() []*wallet.Account`?